google/cloud/spanner_v1/database.py

```diff
@@ -726,6 +730,15 @@ def execute_pdml():
 
                 return result_set.stats.row_count_lower_bound
 
+            with SessionCheckout(self._pool) as session:
```

Comment:
I am not able to add comment in above line, hence adding comment here
pass `trace_name="CloudSpannerOperation.ExecuteStreamingSql"` at https://github.com/googleapis/python-spanner/blob/054a18658eedc5d4dbecb7508baa3f3d67f5b815/google/cloud/spanner_v1/database.py#L717

Reply:
The code from PR #1241 crept into this PR, I am going to revert it.